cognito-pre-token-generation/src/main/java/no/unit/nva/cognito/service/CustomerDbClient.java

```diff
@@ -0,0 +1,74 @@
+package no.unit.nva.cognito.service;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+import no.unit.nva.cognito.model.CustomerResponse;
+import no.unit.nva.customer.ObjectMapperConfig;
+import no.unit.nva.customer.model.CustomerDb;
+import no.unit.nva.customer.model.CustomerDto;
+import no.unit.nva.customer.model.CustomerMapper;
+import no.unit.nva.customer.service.CustomerService;
+import no.unit.nva.customer.service.impl.DynamoDBCustomerService;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Optional;
+
+public class CustomerDbClient implements CustomerApi {
+
+    public static final String ID_NAMESPACE_ENV = "ID_NAMESPACE";
+    private static final Logger logger = LoggerFactory.getLogger(CustomerDbClient.class);
+    public static final String CUSTOMER_NOT_FOUND_FOR_ORG_NUMBER = "Customer not found for orgNumber={}";
+
+    private CustomerService service;
+    private CustomerMapper mapper;
+
+    @JacocoGenerated
+    public CustomerDbClient() {
+        this(defaultCustomerService(),
+                defaultCustomerMapper()
+        );
+    }
+
+    @JacocoGenerated
+    private static DynamoDBCustomerService defaultCustomerService() {
+        return new DynamoDBCustomerService(
+                AmazonDynamoDBClientBuilder.defaultClient(),
```

Comment:
set Region using the AWS_REGION env variable that exists by default in all Lambdas.

Reply:
Example?

Reply:
```
public static final Region AWS_REGION = setupRegion();
    
    @JacocoGenerated
    private static Region setupRegion() {
        return ENVIRONMENT.readEnvOpt("AWS_REGION").map(Region::of).orElse(Region.EU_WEST_1);
    }


private static AmazonDynamoDB defaultDynamoDbClient() {
        return AmazonDynamoDBClientBuilder
                   .standard()
                   .withRegion(ApplicationConstants.AWS_REGION.id())
                   .build();
    }

Reply:
Done